src/reclaim_python_sdk/utils/session_utils.py

```
@@ -66,4 +66,31 @@ async def update_session(session_id, status):
+            headers={'Content-Type': 'application/json'}
+        )
+
+        res = response.json()
+
+        if response.status_code != 200:
+            error_message = f'Error fetching status URL for sessionId: {session_id}. Status Code: {response.status_code}'
+            logger.info(f'{error_message}\n{res}')
+            raise StatusUrlError(error_message)
+
+        return StatusUrlResponse.from_json(res)
+    except Exception as err:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)